apps/frontend/src/network/users/__tests__/UserProfile.test.tsx

```diff
@@ -184,7 +183,6 @@ it("links to the user's team", async () => {
 });
 
 it("links to the user's team proposal", async () => {
-  disable('UPDATED_ROLE_SECTION');
```

Comment:
we can remove this whole test and the one right bellow
`'does not show a proposal for a user whose team has none'`